### ISSUE TITLE
1.12.0 causes "declare export const is not supported." error

### ISSUE THREAD

**Issue by EvHaus** [joarwilk/flowgen#119]
My project doesn't use `flowgen` directly, but I use `flow-typed` which uses it under-the-hood. After resetting my lock file and pulling the latest versions I'm noticing this error when running flow:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/flowgen/index.js.flow:35:16

declare export const is not supported. Use declare export var instead.

     32│   compiler: Compiler,
     33│ |};
     34│
     35│ declare export const flowgen: Flowgen;
     36│
     37│ declare export default flowgen;
     38│
```

Resetting `flowgen` back to version 1.11.0 via resolutions in my package.json files fixes is:

```js
// package.json
"resolutions": {
    "flowgen": "1.11.0"
}
```

Looks like 1.12.0 introduced some code that isn't compatible with flow anymore.

For others running into this, you can also make this issue go away with this in your `.flowconfig`:

```
[ignore]

.*/node_modules/flowgen/index.js.flow
```

**Comment by mhall58** [comment]
> My project doesn't use `flowgen` directly, but I use `flow-typed` which uses it under-the-hood. After resetting my lock file and pulling the latest versions I'm noticing this error when running flow:
> 
> ```
> Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/flowgen/index.js.flow:35:16
> 
> declare export const is not supported. Use declare export var instead.
> 
>      32│   compiler: Compiler,
>      33│ |};
>      34│
>      35│ declare export const flowgen: Flowgen;
>      36│
>      37│ declare export default flowgen;
>      38│
> ```
> 
> Resetting `flowgen` back to version 1.11.0 via resolutions in my package.json files fixes is:
> 
> ```js
> // package.json
> "resolutions": {
>     "flowgen": "1.11.0"
> }
> ```
> 
> Looks like 1.12.0 introduced some code that isn't compatible with flow anymore.
> 
> For others running into this, you can also make this issue go away with this in your `.flowconfig`:
> 
> ```
> [ignore]
> 
> .*/node_modules/flowgen/index.js.flow
> ```

you save me today, thanks for the quick fix

**Comment by orta** [comment]
This is shipped as 1.12.1 - thanks folk
